packages/cadl-rlc-test/test/contoso/cadl-output/2022-11-01-preview/openapi.json

```diff
@@ -458,10 +454,7 @@
           "description": "The faked shared model."
         }
       },
-      "description": "A widget.",
-      "required": [
-        "manufacturerId"
-      ]
```

Comment:
why is this getting removed ?

Reply:
I guess this is a fix in openapi, because this model is only used in patch method. there is no reason to add any required parameter